Zend/zend_compile.c

```diff
@@ -6109,6 +6109,11 @@ static void zend_compile_declare(zend_ast *ast) /* {{{ */
 				zend_error_noreturn(E_COMPILE_ERROR, "Encoding declaration pragma must be "
 					"the very first statement in the script");
 			}
+
+			if (ast->child[1] != NULL) {
+				zend_error_noreturn(E_COMPILE_ERROR, "Encoding declaration pragma must not "
```

Comment:
```suggestion
				zend_error_noreturn(E_COMPILE_ERROR, "Encoding declaration must not "
```
I know that pragma is used in the current wording, but the strict_type one does not use this word, so it might make sense to change the wording to be consistent